docs/source/common/checkpointing.rst

```diff
@@ -385,3 +385,14 @@ Custom Checkpoint IO Plugin
 .. note::
 
     Some ``TrainingTypePlugins`` like ``DeepSpeedStrategy`` do not support custom ``CheckpointIO`` as checkpointing logic is not modifiable.
+
+-----------
+
+***************************
+Managing Remote Filesystems
+***************************
+
+Lightning supports saving and loading checkpoints from a variety of filesystems, including local filesystems and several cloud storage providers
+such as ``s3`` on AWS, ``gcs`` on Google Cloud, or ``adl`` on Azure.
+
+Checkout :ref:`remote_fs` doc for more info.
```

Comment:
```suggestion
Check out the :ref:`remote_fs` documentation for more info.
```

note, the displayed text here would change if the title in the doc changes too, I think. Better to put a custom display text imo

Reply:
good point!